module/assets/helper.js

```diff
@@ -0,0 +1,786 @@
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper");
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+const userExtensionsHelper = require(MODULES_BASE_PATH +
+  "/user-extension/helper");
+const userRolesHelper = require(MODULES_BASE_PATH + "/user-roles/helper");
+const kafkaProducersHelper = require(ROOT_PATH + "/generics/kafka/producers");
+const programUsersHelper = require(MODULES_BASE_PATH + "/programUsers/helper");
+
+module.exports = class AssetsHelper {
+  /**
+   * Get users assets based on Type .
+   * @method
+   * @name fetchAssets
+   * @param {String} queryData -  query for type data.
+   * @param {Object} [bodyData] - request body data data.
+   * @returns {Promise} returns a promise.
+   */
+  static fetchAssets(queryData, bodyData) {
```

Comment:
@praveenKDass - Pass only the required values from body and queryData as specific arguments in this function, e.g. orgId and userId